Sources/MeiliSearch/Indexes.swift

```diff
@@ -346,9 +352,10 @@ public struct Indexes {
    */
   public func getDocument<T>(
     _ identifier: String,
+    fields: [String]? = nil,
     _ completion: @escaping (Result<T, Swift.Error>) -> Void)
   where T: Codable, T: Equatable {
-    self.documents.get(self.uid, identifier, completion)
+    self.documents.get(self.uid, identifier, fields: fields, completion)
```

Comment:
Should this not be added in the `Documents` PR ? (I can close my eyes if you want 😂)

Reply:
dammit, I forgot that yes close your eyes ahhaha!